IBAnimatable/MaskDesignable.swift

```diff
@@ -97,6 +104,24 @@ public extension MaskDesignable where Self: UIView {
     return CGPoint(x: radius * cos(angle) + offset.x, y: radius * sin(angle) + offset.y)
   }
   
+  // MARK: Polygon
+  
+  private func maskPolygonBezierPath(degree: Int) -> UIBezierPath {
```

Comment:
I don't think it should be. That method is just creating the path before drawing it... why do you want it public?